SgmlTests/LoggingXmlReader.cs

```diff
@@ -20,11 +20,11 @@ namespace SGMLTests {
     public class LoggingXmlReader : XmlReader {
 
         //--- Fields ---
-        private readonly XmlReader _reader;
+        private readonly SgmlReader _reader;
         private readonly TextWriter _logger;
 
         //--- Constructors ---
-        public LoggingXmlReader(XmlReader reader, TextWriter logger) {
+        public LoggingXmlReader(SgmlReader reader, TextWriter logger) {
```

Comment:
> Why the change to LoggingXmlReader it was more general taking an XmlReader, why did you change it to take an SgmlReader?

When I was refactoring my test I was thinking of allowing tests to access the `SgmlReader` directly in their callback so they could configure it themselves - but then I saw that the `LoggingXmlReader` doesn't expose the `SgmlReader` so I didn't go further, and I forgot to undo this change.

Reply:
Ok, please undo it, thanks.